styles.css

```diff
@@ -112,3 +112,532 @@ h1 {
+  padding-top: 0;
+  padding-left: 16px;
+  padding-bottom: 10px;
+  padding-right: 20px;
+  background: #98dede;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.3334px;
+  line-height: 80%;
+  color: #1a2236;
+}
+
+.item3 {
+  width: 87px;
+  height: 20px;
+  left: 99px;
+  top: 992px;
+  background: #98dede;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.3334px;
+  line-height: 80%;
+  color: #1a2236;
+}
+
+.item4 {
+  width: 87px;
+  height: 20px;
+  left: 99px;
+  top: 992px;
+  background: #98dede;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 10.3334px;
+  line-height: 80%;
+  color: #1a2236;
+}
+
+.item5 {
+  grid-column-start: 1;
+  grid-column-end: 4;
+  width: 264px;
+  height: 30px;
+  left: 20px;
+  top: 960px;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 16px;
+  line-height: 80%;
+  color: #fff;
+  border: 2px solid #fff;
+}
+
+.works1 {
+  background-color: #1a2236;
+  width: 320px;
+  height: 156px;
+  left: 0;
+  top: 940px;
+  text-align: center;
+  padding-top: 10px;
+  padding-left: 0;
+  justify-content: center;
+  column-gap: 10px;
+  row-gap: 15px;
+}
+
+.works1 > div {
+  text-align: center;
+  font-size: normal;
+  justify-content: center;
+  list-style-type: none;
+}
+
+.item6 {
+  grid-column-start: 1;
+  grid-column-end: 4;
+  text-align: center;
+  margin-left: 15px;
+  margin-top: 15px;
+  width: 264px;
+  height: 16px;
+  left: 20px;
+  top: 960px;
+  font-style: normal;
+  font-weight: 300;
+  font-size: 20px;
+  line-height: 80%;
+  color: #fff;
+}
+
+li {
+  display: inline-flex;
+  flex-direction: row;
+  justify-content: space-between;
+}
+
+.item7 {
+  display: inline-block;
+  flex-direction: row;
+  align-self: center;
+  width: 59px;
+  height: 20px;
+  padding-top: 8px;
+  padding-right: 8px;
+  padding-bottom: 0;
+  background: #98dede;
+  font-weight: 300;
+  font-size: 10.3334px;
+  color: #1a2236;
+  margin-right: 15px;
+  margin-left: 0;
+}
+
+.item8 {
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  width: 27px;
+  height: 20px;
+  padding: 4px 10px;
+  background: #98dede;
+  font-weight: 300;
+  font-size: 10.3334px;
+  color: #1a2236;
+  margin-right: 15px;
+  margin-left: 0;
+}
+
+.item9 {
+  display: inline-flex;
+  flex-direction: row;
+  align-items: center;
+  text-align: center;
+  width: 57px;
+  height: 20px;
+  padding: 4px 10px;
+  background: #98dede;
+  font-weight: 300;
+  font-size: 10.3334px;
+  color: #1a2236;
+  margin-right: 22px;
+  margin-left: 0;
+}
+
+.item10 {
+  grid-column-start: 1;
+  grid-column-end: 4;
+  padding-top: 10px;
+  padding-bottom: 10px;
+  margin-right: 20px;
+  margin-left: 15px;
+  margin-bottom: 15px;
+  align-self: center;
+  border: 2px solid #fff;
+}
+
+.about2 {
+  font-style: normal;
+  font-weight: 400;
+  font-size: 17px;
+  line-height: 21px;
+  letter-spacing: -0.01em;
+  padding-top: 0;
+  padding-right: 150px;
+  padding-bottom: 20px;
+  padding-left: 16px;
+}
```

Comment:
- [ ] I would kindly suggest you work on the `About` section of your project so that it looks exactly like what we have in the Figma design. Currently the `font size`, and `font family` does not correlate with what we have in the Figma design. You can achieve the font family issue by importing the font family name of the design template from google by visiting this website 👉 [Link](https://fonts.google.com/specimen/Poppins?query=Poppins)

Your design 👇 

![mel](https://user-images.githubusercontent.com/78634841/213315750-bf6d0487-b819-4e8a-8108-6227bf319c97.png)

Figma design 👇 

![temp5-real1](https://user-images.githubusercontent.com/78634841/213315804-5d650bc7-1a68-43ac-bb03-e0c6bf472385.png)